code/modules/mining/ores_coins.dm

```diff
@@ -349,6 +349,10 @@
 		var/oldloc = loc
 		sleep(15)
 		if(loc == oldloc && user && !user.incapacitated())
-			user.visible_message("[user] has flipped [src]. It lands on [coinflip].", \
-								 "<span class='notice'>You flip [src]. It lands on [coinflip].</span>", \
-								 "<span class='italics'>You hear the clattering of loose change.</span>")
\ No newline at end of file
+			if(string_attached) //does the coin have a wire attached

```

Comment:
You should do this between lines 342 and 343, because it'll still do the animation otherwise.